tools/static-code-analysis/checkstyle/suppressions.xml

```diff
@@ -27,4 +27,5 @@
     <suppress files=".+[\\/]mail[\\/].+[\\/]MailBuilderTest\.java" checks="ForbiddenPackageUsageCheck"/>
     <suppress files=".+[\\/]mail[\\/].+[\\/]SMTPHandler\.java" checks="ForbiddenPackageUsageCheck"/>
     <suppress files=".+[\\/]mail[\\/].+[\\/]SendMailActions\.java" checks="ForbiddenPackageUsageCheck"/>
+    <suppress files=".+[\\/]proto[\\/].+\.java" checks="[a-zA-Z0-9]*"/>
```

Comment:
That's a leftover trying to get proto java files out of checkstyle because they messed it totally up. Discussed in comment [here ](https://github.com/openhab/openhab-addons/pull/15628#issuecomment-1826284232)so file resetted to origin